.config/i3/lock.sh

```diff
@@ -4,23 +4,22 @@
 timeout="10000"
 
 # Take a screenshot of every screen available:
+scrot -m -z /tmp/lock.png
 xdpyinfo -ext XINERAMA | sed '/^  head #/!d;s///' |
 {
+  cmd="convert /tmp/lock.png"
+	cmdend=""
   while IFS=' :x@,' read i w h x y; do
-    INDEX=$i
-    import -window root -crop ${w}x$h+$x+$y /tmp/head_$i.png
+    cmd+=" -region ${w}x$h+$x+$y -paint 1 -swirl 360"
+    xc=$((x+(w/2)-79))
```

Comment:
The width of the .config/i3/lock.png is hardcoded to 79 pixels. I guess it would be more elegant to check the width of the image but I didn't have time for this.